Userland/Libraries/LibGfx/ImageFormats/WebPWriterLossless.cpp

```diff
@@ -383,14 +377,33 @@ static ErrorOr<NonnullRefPtr<Bitmap>> maybe_write_color_indexing_transform(Littl
     HashMap<ARGB32, u8> color_index_map;
     for (unsigned i = 0; i < color_table_size; ++i)
         color_index_map.set(colors[i], i);
-    auto new_bitmap = TRY(Bitmap::create(BitmapFormat::BGRx8888, bitmap->size()));
+
+    // "When the color table is small (equal to or less than 16 colors), several pixels are bundled into a single pixel.
+    //  The pixel bundling packs several (2, 4, or 8) pixels into a single pixel, reducing the image width respectively."
```

Comment:
Might be nice to add that simple smiley test image as a size regression test?

Reply:
It looks like so: ![smile](https://github.com/SerenityOS/serenity/assets/3487/1b3fcd4a-c274-450a-aa14-3300b02f75b7)

The test a few commits later does test this transform. I tweaked it to also check that compressing with the index transform disabled produces larger files.